server/historian/packages/historian-base/src/routes/utils.ts

```diff
@@ -25,57 +31,7 @@ import {
 } from "../services";
 import { containsPathTraversal, parseToken } from "../utils";
 
-/**
- * Helper function to handle a promise that should be returned to the user.
- * TODO: Replace with handleResponse from services-shared.
- * @param resultP - Promise whose resolved value or rejected error will send with appropriate status codes.
- * @param response - Express Response used for writing response body, headers, and status.
- * @param allowClientCache - sends Cache-Control header with maximum age set to 1 yr if true or no store if false.
- * @param errorStatus - Overrides any error status code; leave undefined for pass-through error codes or 400 default.
- * @param successStatus - Status to send when result is successful. Default: 200
- * @param onSuccess - Additional callback fired when response is successful before sending response.
- */
-export function handleResponse<T>(
-	resultP: Promise<T>,
-	response: Response,
-	allowClientCache?: boolean,
-	errorStatus?: number,
-	successStatus: number = 200,
-	onSuccess: (value: T) => void = () => {},
-) {
-	resultP
-		.then((result) => {
-			if (allowClientCache === true) {
-				response.setHeader("Cache-Control", "public, max-age=31536000");
```

Comment:
nice! does the implementation in services-shared also takes care of setting these headers like Cache-Control, Content Encoding, etc?

Reply:
Yup! I double checked and the implementations are the exact same.